composer.json

```diff
@@ -12,7 +12,8 @@
         "psr/http-server-middleware" : "*",
         "laminas/laminas-diactoros" : "~1.8.4",
         "ext-json": "*",
-        "bjeavons/zxcvbn-php": "^0.4"
+        "bjeavons/zxcvbn-php": "^0.4",
+        "ext-openssl": "*"
```

Comment:
This should not be adding a new dependency to LORIS, and it looks like the only reason it's adding it is in order to do something that MySQL already has a built in function for that can be called instead.

Reply:
@driusan not sure how that happened here.

Reply:
I think it was added for the UUID generation.